src/connectors/openml/openml_dataset_connector.py

```diff
@@ -0,0 +1,101 @@
+"""
+This module knows how to load an OpenML object based on its AIoD implementation,
+and how to convert the OpenML response to some agreed AIoD format.
+"""
+import requests
+from fastapi import HTTPException
+from pydantic import Extra
+from pydantic_schemaorg.DataCatalog import DataCatalog
+from pydantic_schemaorg.DataDownload import DataDownload
+from pydantic_schemaorg.Dataset import Dataset
+from pydantic_schemaorg.QuantitativeValue import QuantitativeValue
+
+from connectors.abstract.dataset_connector import DatasetConnector
+from connectors.platforms import Platform
+from database.models import DatasetDescription
+
+for obj in (DataCatalog, DataDownload, Dataset, QuantitativeValue):
+    obj.Config.extra = Extra.forbid  # Throw exception on unrecognized fields
+
+
+class OpenMlDatasetConnector(DatasetConnector):
+    def platform(self) -> Platform:
+        return Platform.openml
```

Comment:
Would it be reasonable to implement this on the base class by using introspection? Given you expect a certain classname convention, matching on `self.__class__.__name__` should be easy.